test/e2e/versionskew/versionskew.go

```diff
@@ -39,6 +39,8 @@ import (
 const (
 	// base is the release branch used for version skew testing. Empty if none.
 	base = "0.9"
+	// Expected release version(used by the operator)
+	current = "1.0"
```

Comment:
This assumes that the next version will be 1.0.0, right? How does it work given that this version doesn't exist yet?

I'm giving up on reminding you to add a space in English prose before an opening bracket...

Reply:
> This assumes that the next version will be 1.0.0, right? How does it work given that this version doesn't exist yet?

Its need not exist. This is the version number used for the 'operator bundle' generated for the devel code, The bundle gets generated by the 'upgrade test'. And it assumes that the operator gets an upgrade from the 'base'(released) version to the 'current'(future) version.